### PR TITLE
C lightning support

### PR DESCRIPTION
**Pull request by hosiawak**
I've added support for C-Lightning and:

1. Introduced new `--client` and `--client-args` options
2. `--client` can be `LND` or `C-Lightning`
3. `--client-args` - I needed to pass additional options to `lightning-cli`
4. Fixed remote fee calculation in `clnclient.py` (not in `lndclient.py`)
5. Added a new `opener` column (for both LND and C-LN)
6. Updated the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ Tool for pretty printing and optimizing Lightning Network channels.
 
 ![screenshot](screenshot.png)
 
+## Installation
+
+1. Install [poetry](https://python-poetry.org/)
+2. `poetry install`
+3. `poetry run python ./suez`
```
Comment (prusnak) [inline README.md]: This should suffice:

```suggestion
3. `poetry run ./suez`
```

File: README.md [hunk 2]
```diff
@@ -12,7 +18,17 @@ The steepness is controlled via the `fee_sigma` argument.
 
 ## Lightning node support
 
-Currently, Suez only supports LND, but adding support for c-lightning and Eclair should be trivial.
+Currently, Suez supports LND and C-Lightning.
+By default it uses LND (`lncli`).
+You can use it with C-Lightning as follows:
+
+`poetry run python ./suez --client=C-Lightning`
```
Comment (prusnak) [inline README.md]: See above

```suggestion
`poetry run ./suez --client=C-Lightning`
```

File: README.md [hunk 3]
```diff
@@ -12,7 +18,17 @@ The steepness is controlled via the `fee_sigma` argument.
...
+
+If you need to pass additional options to the lncli/lightning-cli you can do so:
+
+`poetry run python ./suez --client=C-Lightning --client-args=--conf=/usr/local/etc/lightningd-bitcoin.conf`
```
Comment (prusnak) [inline README.md]: See above

```suggestion
`poetry run ./suez --client=C-Lightning --client-args=--conf=/usr/local/etc/lightningd-bitcoin.conf`
```

File: suez.py [hunk 4]
```diff
@@ -37,14 +37,21 @@ def _since(ts):
...
+@click.option("--client", default="LND", type=click.Choice(['LND', 'C-Lightning'], case_sensitive=False), help="Type of LN client (LND, C-Lightning, Eclair")
+@click.option("--client-args", default="", help="Extra arguments to pass to client RPC")
+
+def suez(base_fee, fee_rate, fee_sigma, time_lock_delta, client, client_args):
+    clients = {
+        "LND": lambda client_args: LndClient(client_args),
+        "C-Lightning": lambda client_args: ClnClient(client_args),
+    }
+
+    ln = clients.get(client)(client_args)
```
Comment (prusnak) [inline suez.py]:
```suggestion
    ln = clients[client](client_args)
```

File: suez.py [hunk 5]
```diff
@@ -63,7 +70,8 @@ def suez(base_fee, fee_rate, fee_sigma, time_lock_delta):
     table.add_column("last\nforward\n(days)", justify="right")
     table.add_column("local\nfees\n(sat)", justify="right", style="bright_cyan")
     table.add_column("remote\nfees\n(sat)", justify="right", style="bright_cyan")
-    table.add_column("\nalias", max_width=20, no_wrap=True)
+    table.add_column("opener", justify="right")
```
Comment (prusnak) [inline suez.py]:
```suggestion
    table.add_column("\nopener", justify="right")
```

File: suez.py [hunk 6]
```diff
@@ -79,7 +87,10 @@ def suez(base_fee, fee_rate, fee_sigma, time_lock_delta):
             + ("·" * send)
             + "[/green]"
         )
-        uptime = 100 * c.uptime // c.lifetime
+        if hasattr(c, "uptime"):
+            uptime = 100 * c.uptime
+        else:
+            uptime = None
```
Comment (prusnak) [inline suez.py]:
```suggestion
            uptime = "n/a"
```

**Review by prusnak — CHANGES_REQUESTED** [review]
Thanks for your contribution.

Added several suggestions

**Comment by hosiawak** [comment]
@prusnak - Thank you, I'm not a Python programmer (but I like it so far :) ). I'll make these changes soon.

**Comment by prusnak** [comment]
Great! Thanks!

**Pull request merged**